apps/angular/src/app/features/authentification/login/login.component.html

```diff
@@ -0,0 +1,37 @@
+<camp-auth>
+	<form [formGroup]="loginForm" (ngSubmit)="onSubmit()" class="login__form">
+
+		<!-- Email field -->
+		<mat-form-field subscriptSizing="dynamic" appearance="outline" class="login__input-field">
+			<mat-label>Email</mat-label>
+			<input matInput formControlName="email" type="email" >
```

Comment:
I recommend using another way to use reactive forms in the template.
```suggestion
			<input matInput [formControl]="loginForm.controls.email" type="email" >
```
It guarantees us, if our control name will change, VS Code helps us to show error in the HTML template, otherwise, we can build the app without any errors, but, actually, our code is broken.